### PR TITLE
Update cellprofiler-core to 4.2.5

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`cellprofiler-core`](https://bioconda.github.io/recipes/cellprofiler-core/README.html): **4.2.4** &rarr; **4.2.5**

[install with bioconda](http://bioconda.github.io/recipes/cellprofiler-core/README.html) [Conda](https://anaconda.org/bioconda/cellprofiler-core/files)

Info | Link or Description
-----|--------------------
Recipe | [`recipes/cellprofiler-core`](https://github.com//bioconda/bioconda-recipes/tree/bump/cellprofiler_core/recipes/cellprofiler-core) (click to view/edit other files)
Summary | Dependency for CellProfiler v4
Home | [https://github.com/CellProfiler/core](https://github.com/CellProfiler/core)
Releases |[https://pypi.org/pypi/cellprofiler-core/json](https://pypi.org/pypi/cellprofiler-core/json)
Recipe Maintainer(s) | @raivivek

***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

File: recipes/cellprofiler-core/meta.yaml [hunk 1]
```diff
@@ -22,15 +22,15 @@ requirements:
     - boto3 >=1.12.28
     - centrosome ==1.2.1
     - docutils ==0.15.2
-    - h5py ~=3.6.0
+    - h5py >3.6.0
```
Comment (priesgo) [inline recipes/cellprofiler-core/meta.yaml:25]: The right translation of `~=3.6.0` is `>=3.6.0,<3.7.0`. Same applies for other dependencies

**Review by priesgo — CHANGES_REQUESTED** [review]
See comment on dependencies

**Comment by priesgo** [comment]
ping @raivivek

**Comment by raivivek** [comment]
@priesgo Would you be interesting in adding a commit to this PR addressing your requested change?

**Comment by priesgo** [comment]
Uh, what happened there?

**Pull request closed without merging**